#PointCloud input pointcloud
#sensor_msgs/PointCloud2 input
---
#Pose of centroid
geometry_msgs/PoseStamped output
